### PR TITLE
WIP added elastic pooling to SQL server

### PR DESCRIPTION
**Pull request by Renetnielsen**
## References

* #0000

### CODE REVIEW

File: build/infrastructure/main/mssql-data.tf [hunk 1]
```diff
@@ -35,6 +35,12 @@ module "mssql_data" {
       end_ip_address    = "255.255.255.255"
     }
   ]
+  elastic_pool_sku              = {
+    name      = "GP_Gen4"
+    tier      = "GeneralPurpose"
+    family    = "Gen4"
+    capacity  = 4
```
Comment (djorgensendk) [inline build/infrastructure/main/mssql-data.tf:42]: In the TF module capacity is of type string. Should it be number instead?

File: build/infrastructure/main/mssql-data.tf [hunk 2]
```diff
@@ -16,7 +16,7 @@ locals {
 }
 
 module "mssql_data" {
-  source                        = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//azure/mssql-server?ref=5.7.0"
+  source                        = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//azure/mssql-server?ref=renetnielsen/elastic-pooling"
```
Comment (djorgensendk) [inline build/infrastructure/main/mssql-data.tf:19]: Must be changed before merge

**Review by djorgensendk — CHANGES_REQUESTED** [review]

File: build/infrastructure/main/mssql-data.tf (hunk 1 above)
Comment (Renetnielsen) [reply build/infrastructure/main/mssql-data.tf:42]: Fixed

File: build/infrastructure/main/mssql-data.tf (hunk 2 above)
Comment (Renetnielsen) [reply build/infrastructure/main/mssql-data.tf:19]: Yup, can't change it before i can release the module. 
Which is when we have the backup strategy (or in conjunction with our VNET rollout)

**Comment by Renetnielsen** [comment]
@cla-bot check

**Comment by kerbou** [comment]
Closing PR, it's 6 months old

**Pull request closed without merging**